src/main/kotlin/app/revanced/patches/duolingo/unlocksuper/fingerprints/DuolingoSuperFingerprint.kt

```diff
@@ -0,0 +1,13 @@
+package app.revanced.patches.duolingo.unlocksuper.fingerprints
+
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import com.android.tools.smali.dexlib2.AccessFlags
+
+object DuolingoSuperFingerprint : MethodFingerprint(
```

Comment:
Give this fingerprint a name that resembles the method it resolves to. For example a fingerprint of a method that checks a login can be called "CheckLoginFingerprint".